apps/pesayetu/src/components/ExplorePage/index.js

```diff
@@ -8,8 +9,8 @@ import useExplore from "./useExplore";
 import useProfileGeography from "./useProfileGeography";
 import useStyles from "./useStyles";
 
-import Location from "@/pesayetu/components/HURUmap/Location";
 import Panel from "@/pesayetu/components/HURUmap/Panel";
+import theme from "@/pesayetu/theme";
```

Comment:
You can use `useTheme` hook at component component level or `sx=((theme) => (` function instead of hard-coding paths to theme.